### PR TITLE
Testing Streamlink Server

### PR DESCRIPTION
**Pull request by jkemp814**
Testing Streamlink server. Please advise.

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
Works perfectly on my side, however watch out. The australian ABC and the american ABC are 2 different companies/societies AFAIK

**Comment by jkemp814** [comment]
Yes. This is the channel I wanted but didn't want to go to YouTube all the time. Seen it wasn't in the playlist.

File: channels/us.m3u [hunk 1]
```diff
@@ -1854,3 +1854,5 @@ https://thegateway.app/YouToo/CueTones/playlist.m3u8
 https://thegateway.app/YouToo/YTamerica/playlist.m3u8
 #EXTINF:-1 tvg-id="KanalDisney.us" tvg-country="RU" tvg-language="Russian" tvg-logo="https://i.imgur.com/Q9KoVy9.png" group-title="Kids",Канал Disney (576p) [Not 24/7]
 http://188.40.68.167/russia/disney/playlist.m3u8
+#EXTINF:-1 tvg-id="ABCNewsAustralia.us" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/bLzWAtX.jpg" group-title="News",ABC News (Australia)
+https://ask-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/watch?v=W1ilCy6XrmI/live
```
Comment (freearhey) [inline channels/us.m3u:1858]: image

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/us.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/us.m3u:1858]: 429 Too Many Requests. It seems that someone tried too hard.

Comment (jkemp814) [reply channels/us.m3u:1860]: Yeah, I'm getting the same error. Is that coming from Youtube, or Heruko?

Comment (jkemp814) [reply channels/us.m3u:1860]: Seems the links I tested going to YouTube from my Heroku app IP, will not work. 
When I use another URL's it does. Can someone confirm with a link other than YouTube?

**Pull request closed without merging**